4-React/src/components/DogList/DogList.jsx

```
@@ -0,0 +1,11 @@
+import { Dog } from '../Dog'
+
+export function DogList({ arrDogs }) {
+  return (
+    <>
+      {arrDogs.map((dog) => (
+        <Dog name={dog.name} breed={dog.breed} superpower={dog.superpower} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)